src/drivers/indexeddb.js

```diff
@@ -775,6 +794,7 @@ function clear(callback) {
             })
             .catch(reject);
     });
+    promise.then(unref, unref);
```

Comment:
The current solution will retain the transactions in memory forever and never free them. This is because when `promise.then(unref, unref);` is executed, `unref` is still `undefined` since the callback hasn't executed yet.

To get around this, you could move the `promise.then(unref, unref);` inside the callback, or wrap the unref calls inside another function, e.g. something like:

```
promise.then(
  () => unref && unref(),
  () => unref && unref()
);
```